### PR TITLE
Update README links to the examples directory

### PR DESCRIPTION
**Pull request by frightenedmonkey**
What it says on the tin. No code updates.

### CODE REVIEW

**Comment by chris-rock** [comment]
awesome @frightenedmonkey

File: README.md [hunk 1]
```diff
@@ -112,7 +112,11 @@ describe yaml('.kitchen.yml') do
 end
 ```
 
-Also have a look at our [example](https://github.com/chef/inspec/tree/master/examples/test-kitchen) that uses `inspec` in combination with `test-kitchen`
+Also have a look at our examples for:
+- [Usiing inspec with test-kitchen & chef](https://github.com/chef/inspec/tree/master/examples/kitchen-chef)
```
Comment (kellydunn) [inline README.md:116]: @frightenedmonkey typo here `Using` instead of `Usiing`

**Comment by chris-rock** [comment]
@kellydunn fixed with #382

**Comment by kellydunn** [comment]
:+1:

**Pull request merged**